docs/index.bs

```diff
@@ -1862,6 +1862,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
             1. If |r|'s [=request/url=]'s [=url/scheme=] is not one of "<code>http</code>" and "<code>https</code>", then:
                 1. [=fetch/Terminate=] all the ongoing <a>fetches</a> initiated by |requests| with reason *fatal*.
                 1. Break the loop.
+            1. If |r|'s [=request/client=]'s [=environment settings object/global object=] is a {{ServiceWorkerGlobalScope}} object, set |request|'s [=skip-local-service-worker flag=].
```

Comment:
Are you introducing a new flag here? Or did you intend to use _skip-service-worker flag_?